code/modules/mob/living/simple_animal/friendly/snake.dm

```diff
@@ -1,38 +1,38 @@
 /mob/living/simple_animal/hostile/retaliate/poison
-    var/poison_per_bite = 0
-    var/poison_type = "toxin"
+	var/poison_per_bite = 0
+	var/poison_type = "toxin"
 
 /mob/living/simple_animal/hostile/retaliate/poison/AttackingTarget()
-    . = ..()
-    if(. && isliving(target))
-        var/mob/living/L = target
-        if(L.reagents && !poison_per_bite == 0)
-            L.reagents.add_reagent(poison_type, poison_per_bite)
-        return .
+	. = ..()
+	if(. && isliving(target))
+		var/mob/living/L = target
+		if(L.reagents && !poison_per_bite == 0)
+			L.reagents.add_reagent(poison_type, poison_per_bite)
+		return .
 
 /mob/living/simple_animal/hostile/retaliate/poison/snake
-        name = "snake"
-        desc = "A slithery snake. These legless reptiles are the bane of mice and adventurers alike."
-        icon_state = "snake"
-        icon_living = "snake"
-        icon_dead = "snake_dead"
-        speak_emote = list("hisses")
-        health = 20
-        maxHealth = 20
-        attacktext = "bites"
-        melee_damage_lower = 5
-        melee_damage_upper = 6
-        response_help  = "pets"
-        response_disarm = "shoos"
-        response_harm   = "steps on"
-        faction = list("hostile")
-        ventcrawler = VENTCRAWLER_ALWAYS
-        density = FALSE
-        pass_flags = PASSTABLE | PASSMOB
-        mob_size = MOB_SIZE_SMALL
-        gold_core_spawnable = FRIENDLY_SPAWN
-        obj_damage = 0
-        environment_smash = ENVIRONMENT_SMASH_NONE
+	name = "snake"
+	desc = "A slithery snake. These legless reptiles are the bane of mice and adventurers alike."
+	icon_state = "snake"
+	icon_living = "snake"
+	icon_dead = "snake_dead"
+	speak_emote = list("hisses")
+	health = 20
+	maxHealth = 20
+	attacktext = "bites"
+	melee_damage_lower = 5
+	melee_damage_upper = 6
+	response_help  = "pets"
+	response_disarm = "shoos"
+	response_harm   = "steps on"
```

Comment:
"stomps on"?

Steps seems a little... unharmful. Unless I don't know what this var is for.

Reply:
https://ih1.redbubble.net/image.267700561.9566/flat,800x800,070,f.u1.jpg